main.js

```diff
@@ -34,10 +38,16 @@ var pastActivities = [];
 //Add event listeners below
 categoryBoxWrapper.addEventListener('click', activateCategory);
 startActivityButton.addEventListener('click', submitForm);
-
 startTimerButton.addEventListener('click', function() {
   currentActivity.startTimer();
```

Comment:
Once start is pressed, I suggest you also disable the start button. Currently, if you press it again when the timer is running, you get multiple timers running at once & it causes a stutter with the timer display

Reply:
The Distortion! We are working through this solution currently and will address it in a future push/merge!